traitlets/traitlets.py

```diff
@@ -628,16 +628,15 @@ def __init__(self, *args, **kw):
                 setattr(self, key, value)
 
     @contextlib.contextmanager
-    def hold_trait_notifications(self):
+    def mute_trait_notifications(self):
         """Context manager for bundling trait change notifications and cross
         validation.
-
         Use this when doing multiple trait assignments (init, config), to avoid
         race conditions in trait notifiers requesting other trait values.
         All trait notifications will fire after all values have been assigned.
```

Comment:
The docstring should probably be modified to reflect what `mute_trait_notifications` does.